sale_coupon_limit/static/description/icon.svg

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Shouldn't be this icon a number and let this icon to incompatibility?

Reply:
The incompatibility one is better :slightly_smiling_face: